webhook_to_fedora_messaging/endpoints/util.py

```diff
@@ -0,0 +1,60 @@
+from flask import Response, Request
+from ..config.defaults import FASJSON_URL
+import fasjson_client
+
+
+def not_found() -> Response:
+    return Response({'message': 'Not Found'}, status=404, mimetype='application/json')
+
+
+def success(data: dict) ->Response:
+    return Response(data, status=200, mimetype='application/json')
+
+
+def bad_request() -> Response:
+    return Response("{'message': 'Bad Request'}", status=400, mimetype='application/json')
+
+
+def created(data: dict) -> Response:
+    return Response(data, status=201, mimetype='application/json')
+
+
+def conflict(data: dict) -> Response:
+    return Response(data, status=409, mimetype='application/json')
+
+
+def validate_request(request: Request, fields=['username']):
+    return all(field in request for field in fields)
+
+    
+def verify_signature(payload_body: str, secret_token: str, signature_header: str) -> bool:
+    """Verify that the payload was sent from GitHub by validating SHA256.
+
+    Raise and return 403 if not authorized.
+
+    Args:
+        payload_body: original request body to verify (request.body())
+        secret_token: GitHub app webhook token (WEBHOOK_SECRET)
+        signature_header: header received from GitHub (x-hub-signature-256)
+    """
+    if not signature_header:
+        return False
+    hash_object = hmac.new(secret_token.encode('utf-8'), msg=payload_body, digestmod=hashlib.sha256)
+    expected_signature = "sha256=" + hash_object.hexdigest()
+    
+    if not hmac.compare_digest(expected_signature, signature_header):
+        return False
+    return True
+
+
+def fas_by_github(username: str) -> str:
+    """Get the Fedora Account System Username of the given github username
```

Comment:
GitHub is a proper noun.

```suggestion
    """Get the Fedora Account System Username of the given GitHub username
```